tests/py/requirements.txt

```diff
@@ -1,15 +1,15 @@
 # This file is specifically to install correct version of libraries during CI testing.
-# The index url for torch & torchvision libs is configured in install-torch-tensorrt.sh based on CUDA version
 # networkx library issue: https://discuss.pytorch.org/t/installing-pytorch-under-python-3-8-question-about-networkx-version/196740
+expecttest==0.1.6
+networkx==2.8.8
+numpy<2.0.0
+parameterized>=0.2.0
 pytest>=8.2.1
 pytest-xdist>=3.6.1
-networkx==2.8.8
-torch>=2.4.0.dev,<2.5.0
-torchvision>=0.19.0.dev,<0.20.0
---extra-index-url https://pypi.ngc.nvidia.com
 pyyaml
 tensorrt==10.0.1
+tensorrt-cu12-bindings==10.0.1
```

Comment:
rather not pin these in test, ideally the package brings in the correct version of these deps

Reply:
It's not guaranteed to always get installed though and hence causing `FileNotFoundError: Could not find nvinfer.dll`, like https://github.com/pytorch/TensorRT/actions/runs/9460735888/job/26060910960#step:12:466.

Reply:
It would be better to explicitly list tensorrt-cu12-bindings in the pyproject.toml

Reply:
OK. Let's try.